traits/ctraits.c

```diff
@@ -580,6 +580,39 @@ set_value ( PyObject ** field, PyObject * value ) {
     return 0;
 }
 
+static PyObject *
+get_trait_flag ( trait_object * trait, int mask ) {
+    PyObject * result;
+
+    if ( (trait->flags & mask) == 0 ) {
+        result = Py_False;
```

Comment:
I suggest using the `Py_RETURN_TRUE` convenience macro in this branch (and `Py_RETURN_FALSE` in the next one, of course).

Reply:
Also, let's drop the awful extra-space style in new code, and follow an approximation of PEP 7. (At some point before the 6.0 release, I'd like to run a formatting utility across the whole of this file, but we need to time that carefully to minimise disruption to existing PRs.)